actions/models.py

```diff
@@ -5,7 +5,7 @@
 class Action(models.Model):
     org = models.CharField(max_length=200)
     repo_name = models.CharField(max_length=200)
-    about = models.TextField()
+    about = models.TextField(blank=True)
```

Comment:
It's not what I expect. And I knew the "Don't use `null=True` for string-based fields", too 🤦🏻‍♂️. I don't think I need to set `default`, because I convert `None` to an empty string. However, I'll do so for documentation. I've removed the `blank` change/migration, added the `default` change/migration, and force pushed.